microsoft/testsuites/core/disk_demo.py

```diff
@@ -0,0 +1,69 @@
+from typing import Any
+
+from assertpy import assert_that
+
+from lisa import (
+    Logger,
+    Node,
+    TestCaseMetadata,
+    TestSuite,
+    TestSuiteMetadata,
+    simple_requirement,
+    search_space,
+)
+from lisa.features import Disk
+from lisa.features.disks import DiskStandardHDDLRS
+from lisa.node import Node
+from lisa.schema import DiskType, DiskOptionSettings
+from lisa.tools import Echo, Cat, Mount
+@TestSuiteMetadata(
+    area="first test suite",
+    category="functional",
+    description="""
+    this is an example test suite.
+    it shows a basic sample of writing a test case.
+    it also is a good chance to learn about the testsuites.
+    """,
+)
+class FirstTestSuite(TestSuite):
+    @TestCaseMetadata(
+        description="""
+        Demo writing a test and using the disk.
+        """,
+        priority=4,
+        requirement=simple_requirement(
+            disk=DiskOptionSettings(
+                disk_type=DiskType.StandardHDDLRS,
+                data_disk_count=search_space.IntRange(min=1),
+            )   
+        ),
+    )
+    def check_disk_move_operation_rydailey(self, node: Node, log: Logger,) -> None:
+        # write hello world to a file
+        text = "Hello World!"
```

Comment:
These should be class level constants. Also, please follow the naming conventions from the docs: https://mslisa.readthedocs.io/en/main/write_test/guidelines.html

Reply:
Changed to class level and resolved naming issues